datasets/covid_tweets_japanese/README.md

```diff
@@ -0,0 +1,136 @@
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [COVID-19 日本語Twitterデータセット homepage](http://www.db.info.gifu-u.ac.jp/data/Data_5f02db873363f976fce930d1)
+- **Repository:** [N/A]
+- **Paper:** [N/A]
+- **Leaderboard:** [N/A]
+- **Point of Contact:** Check the homepage.
+
+### Dataset Summary
+
+53,640 Japanese tweets with annotation if a tweet is related to COVID-19 or not. The annotation is by majority decision by 5 - 10 crowd workers. Target tweets include "COVID" or "コロナ". The period of the tweets is from around January 2020 to around June 2020. The original tweets are not contained. Please use Twitter API to get them, for example.
+
+### Supported Tasks and Leaderboards
+
+Text-classification, Whether the tweet is related to COVID-19, and whether it is fact or opinion.
+
+### Languages
+
+The text can be gotten using the IDs in this dataset is Japanese, posted on Twitter.
+
+## Dataset Structure
+
+### Data Instances
+
+CSV file with the 1st column is Twitter ID and the 2nd column is assessment option ID.
+
+### Data Fields
+
+- `tweet_id`: Twitter ID.
+- `assessment_option_id`: The selection result. It has the following meanings:
+  - 63: a general fact: generally published information, such as news.
+  - 64: a personal fact: personal news. For example, a person heard that the next-door neighbor, XX, has infected COVID-19, which has not been in a news.
+  - 65: an opinion/feeling
+  - 66: difficult to determine if they are related to COVID-19 (it is definitely the tweet is not "67: unrelated", but 63, 64, 65 cannot be determined)
+  - 67: unrelated
+  - 68: it is a fact, but difficult to determine whether general facts, personal facts, or impressions (it may be irrelevant to COVID-19 since it is indistinguishable between 63 - 65 and 67).
+
+### Data Splits
+
+[N/A]
+
+## Dataset Creation
+
+### Curation Rationale
+
+N/A because the paper is not yet published.
```

Comment:
```suggestion
[More Information Needed]

## Dataset Creation

### Curation Rationale

[More Information Needed] because the paper is not yet published.
```